<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertNewRecord</class>
 <widget class="QDialog" name="InsertNewRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>863</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Records not filled with details will be ignored.</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Number Of Records To Insert:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbNumberOfRecords"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tblInsert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>821</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbInsertIntoTableName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic</string>
   </property>
   <property name="text">
    <string>Inserting into</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddRecords">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
